docs/html/development/getting-started.rst

```diff
@@ -8,7 +8,61 @@ This document is meant to get you setup to work on pip and to act as a guide and
 reference to the the development setup. If you face any issues during this
 process, please `open an issue`_ about it on the issue tracker.
 
-Development tools
+Running Developent Version
+==========================
+
+Clone git repository and change dir into it:
+
+.. code-block:: console
+
+    $ git clone https://github.com/pypa/pip
+    $ cd pip
+
+Then run `python src/pip`:
```

Comment:
there needs to be a big fat warning here on why this works and that this absolutely doenst work for running the testsuite that ensure pip works ^^

Reply:
The explanation why it works is below. And there is no visible way to run tests without `tox`, so people have to read about `tox` below anyway.